extension/getAllElementsByPath.js

```diff
@@ -84,10 +84,10 @@ function getY(node) {
 
 function addCoordinates(extractedAttributes, node) {
 	// these attributes need special treatment
-	extractedAttributes["absolute-x"] = getX(node) * (WANTED_WIDTH/fullWidth);
-	extractedAttributes["absolute-y"] = getY(node) * (WANTED_WIDTH/fullWidth);
-	extractedAttributes["absolute-width"] = node.getBoundingClientRect().width * (WANTED_WIDTH/fullWidth);
-	extractedAttributes["absolute-height"] = node.getBoundingClientRect().height * (WANTED_WIDTH/fullWidth);
+	extractedAttributes["absolute-x"] = getX(node) * fullWidth;
```

Comment:
I assume we have to adapt this in recheck-web-js too:

https://github.com/retest/recheck-web-js/blob/master/src/script.ts#L50

Somewhere else?

Reply:
Doing that...